src/theme/index.hbs

```diff
@@ -72,13 +78,13 @@
 
                 <!-- Mobile navigation buttons -->
                 {{#previous}}
-                    <a href="{{link}}" class="mobile-nav-chapters previous">
+                    <a rel="prev" href="{{link}}#content" class="mobile-nav-chapters previous">
```

Comment:
Ah I see, focusing on the content rather than the TOC has been requested in #235. But `#content` does not solve that issue, could you maybe try adding `tabindex=1` to the content to see if it solves both issues at the same time? :)